internal/adapters/binance/events.go

```diff
@@ -2,28 +2,31 @@ package binance
 
 import (
 	"fmt"
-	"strconv"
-	"strings"
 
 	"github.com/adshao/go-binance/v2"
+	"github.com/matrixbotio/exchange-gates-lib/internal/adapters/binance/helpers/mappers"
+	"github.com/matrixbotio/exchange-gates-lib/internal/adapters/binance/wrapper"
 	"github.com/matrixbotio/exchange-gates-lib/internal/workers"
 	pkgStructs "github.com/matrixbotio/exchange-gates-lib/pkg/structs"
-	"github.com/matrixbotio/exchange-gates-lib/pkg/utils"
 )
 
+// TBD: https://github.com/matrixbotio/exchange-gates-lib/issues/196
 // PriceWorkerBinance - MarketDataWorker for binance
 type PriceWorkerBinance struct {
 	workers.PriceWorker
+	binanceAPI wrapper.BinanceAPIWrapper
 }
 
 // TradeEventWorkerBinance - TradeEventWorker for binance
 type TradeEventWorkerBinance struct {
 	workers.TradeEventWorker
+	binanceAPI wrapper.BinanceAPIWrapper
 }
 
-// GetPriceWorker - create new market data worker
```

Comment:
Either rename the method, either do not remove the comment, because the name is incorrect.
I prefer the first option